packages/client/hmi-client/src/components/workflow/ops/intervention-policy/tera-intervention-card.vue

```diff
@@ -105,13 +105,14 @@
 						option-value="value"
 						placeholder="Select a trigger"
 					/>
-					<Dropdown
+					crosses the threshold
+					<!-- <Dropdown
```

Comment:
I dont think we plan to introduce this any time soon. safe to remove rather than comment out

Reply:
💯 we have a policy of not keeping code commented out.
We have git if people want to re-use old code.